docs/marks/tip.md

```diff
@@ -79,29 +85,66 @@ Plot.dot(olympians, {
-- the **pointerSize** option controls the visual appearance
-- the **pathFilter** option controls the drop shadow
-- you can use the **textAnchor** option, and other text options
+If a channel is bound to the *color* or *opacity* scale, the tip mark displays a swatch to reinforce the encoding, such as female <span :style="{color: d3.schemeTableau10[0]}">■</span> or male <span :style="{color: d3.schemeTableau10[1]}">■</span>.
+
+The tip mark recognizes that **x1** & **x2** and **y1** & **y2** are paired channels. Below, observe that the *weight* shown in the tip is a range such as 64–66 kg; however, the *frequency* is shown as the difference between the **y1** and **y2** channels. The latter is achieved by the stack transform setting a channel hint to indicate that **y1** and **y2** represent a length.
+
+:::plot defer
+```js
+Plot.rectY(olympians, Plot.binX({y: "count"}, {x: "weight", fill: "sex", tip: true})).plot()
+```
+:::
+
+This even works when stacking negative values, say to mirror the histogram instead of stacking it. (The tip displays negative frequency, but this is consistent with the *y* axis.)
+
+:::plot defer
+```js
+Plot.rectY(olympians, Plot.binX({y: "sum"}, {x: "weight", y: (d) => d.sex === "male" ? 1 : -1, fill: "sex", tip: true})).plot({y: {label: "Frequency"}})
+```
+:::
+
+The tip mark does not provide options for formatting channel names or values. When a channel is bound to a scale, the scale’s label is shown instead of the channel name. If you are interested in this feature, please upvote [#1612](https://github.com/observablehq/plot/issues/1612).
+
+The tip mark supports nine different orientations specified by the **anchor** option: the four sides (*top*, *right*, *bottom*, *left*), the four corners (*top-left*, *top-right*, *bottom-right*, *bottom-left*), and *middle*. Note that when *middle* is used, the tip will obscure its anchor point.
+
+:::plot defer
+```js
+Plot.plot({
+  height: 160,
+  marks: [
+    Plot.frame({strokeOpacity: 0.2}),
+    [
+      "top", "right", "bottom", "left", // sides
+      "top-left", "top-right", "bottom-right", "bottom-left", // corners
+      "middle"
+    ].map((anchor) => [
+      Plot.dot({length: 1}, {frameAnchor: anchor, fill: "blue"}),
+      Plot.tip([anchor], {frameAnchor: anchor, anchor})
+    ])
+  ]
+})
+```
+:::
+
+If you don’t specify an **anchor**, the tip mark will choose one automatically. It will prefer *top-left* if the tip fits; otherwise the it will switch sides to try to contain the tip within the plot’s frame. When dynamically rendering the tip mark, say with the [pointer interaction](../interactions/pointer.md), the tip will also attempt to use the anchor it chose previously, making the tip more stable as you move the pointer and improving readability. In some cases, it may not be possible to fit the tip within the plot’s frame; consider setting the plot’s **style** to `overflow: visible;` to prevent the tip from being truncated.
```

Comment:
```suggestion
If you don’t specify an **anchor**, the tip mark will choose one automatically. It will prefer *top-left* if the tip fits; otherwise it will switch sides to try to contain the tip within the plot’s frame. When dynamically rendering the tip mark, say with the [pointer interaction](../interactions/pointer.md), the tip will also attempt to use the anchor it chose previously, making the tip more stable as you move the pointer and improving readability. In some cases, it may not be possible to fit the tip within the plot’s frame; consider setting the plot’s **style** to `overflow: visible;` to prevent the tip from being truncated.
```